src/wallet/wallet.cpp

```diff
@@ -3523,30 +3589,65 @@ void CReserveKey::ReturnKey()
     vchPubKey = CPubKey();
 }
 
-static void LoadReserveKeysToSet(std::set<CKeyID>& setAddress, const std::set<int64_t>& setKeyPool, CWalletDB& walletdb) {
-    for (const int64_t& id : setKeyPool)
+void CWallet::CheckKeypoolMinSize() {
```

Comment:
Could s/CheckKeypoolMinSize/CheckKeypoolCriticalSize

There was also an old comment about renaming this. https://github.com/bitcoin/bitcoin/pull/10882#discussion_r128611234. Could go with something with something like `MaybeShutdownForLaggingWallet`